utils/build-script-impl

```diff
@@ -2196,6 +2197,13 @@ for host in "${ALL_HOSTS[@]}"; do
                     )
                 fi
 
+                if [[ ! "${SWIFT_ENABLE_EXPERIMENTAL_STRING_PROCESSING}" ]] ; then
```

Comment:
This `if ! ENABLE` looks like it's the opposite of `if ! SKIP` above, is this doing the correct thing?

Reply:
And is `SWIFT_ENABLE_EXPERIMENTAL_STRING_PROCESSING` not being enabled being properly handled in the CMake?

Reply:
Good catch. I think I've fixed it. I moved the flag declaration from swift_build_support/products/swift.py to build-script-impl.